native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -928,12 +921,12 @@ JNIEXPORT jlong JNICALL Java_sage_UIManager_findWindow(
 JNIEXPORT jboolean JNICALL Java_sage_DirectX9SageRenderer_hasDirectX90
   (JNIEnv *env, jclass jc)
 {
-    TCHAR szPath[512];
-    TCHAR szFile[512];
-    if (GetSystemDirectory(szPath, 512) != 0)
+    TCHAR szPath[_MAX_PATH];
+    TCHAR szFile[_MAX_PATH];
+    if (GetSystemDirectory(szPath, _MAX_PATH) != 0)

```

Comment:
Use sizeof(szPath) instead of _MAX_PATH here.

Reply:
Done